tools/pipelines/templates/include-use-node-version.yml

```diff
@@ -4,14 +4,13 @@
 # include-use-node-version template to use a specific node version
 
 parameters:
-# The node version required. The version must already be installed in the image. 
+# The node version required. The version must already be installed in the image.
 - name: version
   type: string
   default: '18.17.1'
 
 steps:
 - task: Bash@3
-  name: SetNodeVersion
```

Comment:
Nothing leverages this step by name, and when using `include-telemetry-setup.yml` it might end up being included again in the pipeline, which causes problems because of steps with duplicate names. So just removed it.